php/class-customize-snapshot-manager.php

```diff
@@ -494,9 +502,6 @@ public function update_snapshot() {
 		if ( ! check_ajax_referer( self::AJAX_ACTION, 'nonce', false ) ) {
 			status_header( 400 );
 			wp_send_json_error( 'bad_nonce' );
-		} elseif ( ! current_user_can( 'customize' ) ) {
```

Comment:
@valendesigns because I added it below instead. The `create_posts` cap is now mapped to `do_not_allow` in the post type so as to hide the “Add New” links for snapshots in the admin. So then we have to use `customize` cap for this instead directly here, which would be doing below anyway.